pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateBSPDummyKitRoute.java

```diff
@@ -31,22 +32,24 @@ public Object handle(Request request, Response response) throws Exception {
         logger.info("Found kitlabel " + kitLabel + " in Mercury request");
         int ddpInstanceId = (int) DBUtil.getBookmark(DUMMY_REALM_NAME);
         logger.info("Found ddp instance id for mock test " + ddpInstanceId);
-        DDPInstance mockDdpInstance = DDPInstance.getDDPInstanceById(ddpInstanceId);
-        if (mockDdpInstance != null) {
-            logger.info("Found mockDdpInstance " + mockDdpInstance.getName());
+        DDPInstanceDto mockDdpInstanceDto =
```

Comment:
if this is dummy route and will never be used, why should we add complexity of saving all this dummy instance names into database and fetching it and etc? we can just mock it here directly using Builder of `DDPInstanceDto`, or I did not get the idea of this route?